### PR TITLE
Perform Dependency Upgrades and System Fixes

### PR DESCRIPTION
**Pull request by lcduong**
Perform the following upgrades and maintenance

Fixes https://github.com/fossasia/eventyay-tickets/issues/130

> * [x]  Replace slimit with https://pypi.org/project/rjsmin/
> * [x]  Upgrade jQuery
> * [x]  PDF: Upgrade fabric.js
> * [x]  Upgrade package-lock to v2
> * [x]  Get rid of unmaintained dependency python-u2flib-server
> * [x]  Upgrade NodeJS
> * [x]  Improve npm install call in pyproject.toml (previously setup.py)
> * [x]  Update webauthn requirement from to ==2.0
> * [x]  Update django-oauth-toolkit requirement to ==2.3.
> * [x]  Upgrade celery to 5.4.*
> * [x]  chardet to 5.2.*
> * [x]  css-inline to 0.14.*
> * [x]  django-bootstrap3 to 24.2
> * [x]  django-compressor to 4.5
> * [x]  django-countries to 7.6.*
> * [x]  django-filter to 24.2
> * [x]  django-formtools to 2.5.1
> * [x]  django-hierarkey to 1.2.*
> * [x]  django-hijack 3.5.*
> * [x]  django-otp	1.5.*
> * [x]  django-phonenumber-field	7.3.*
> * [x]  django-statici18n	2.5.*
> * [x]  djangorestframework	3.15.*
> * [x]  importlib_metadata	8.*
> * [x]  libsass	0.23.*
> * [x]  markdown	3.6
> * [x]  Pillow	10.3.* => 10.4.*
> * [x]  protobuf	5.27.*
> * [x]  pycparser	2.22
> * [x]  python-dateutil	2.9.*
> * [x]  reportlab	4.2.*
> * [x]  stripe	7.9.*
> * [x]  tqdm	4.66.*
> * [x]  webauthn	2.2.*
> * [x]  fakeredis	2.23.*
> * [x]  flake8	7.1.*
> * [x]  isort	5.13.*
> * [x]  pep8-naming	0.14.*
> * [x]  pytest	8.2.*
> * [x]  pytest-mock	3.14.*
> * [x]  pytest-rerunfailures	14.*
> * [x]  pytest-xdist	3.6.*

### CODE REVIEW

**Comment by mariobehling** [comment]
@sourcery-ai review

File: doc/admin/scaling.rst [hunk 1]
```diff
@@ -233,4 +233,4 @@ throughput. If you want to use pretix for an event with 10,000+ tickets that are
 within minutes, please get in touch to discuss possible solutions. We'll work something out for you!
 
 
-.. _object storage cluster: https://behind.pretix.eu/2018/03/20/high-available-cdn/
\ No newline at end of file
+.. _object storage cluster: https://behind.eventyay.com/2018/03/20/high-available-cdn/
```
Comment (mariobehling) [inline doc/admin/scaling.rst]: Do not change this.

**Review by mariobehling — CHANGES_REQUESTED** [review]
Could you please split this PR into two:
a) all system upgrades
b) all changes from pretix.eu to eventyay.com

**Comment by lcduong** [comment]
hi @mariobehling, conflict is solved

**Pull request merged**